### PR TITLE
Changes default path and change getUser route

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/users/routes.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ const UserModel = require('../../utils/schema/usersSchema');
 const multer = require('multer');
 const csv = require('csvtojson');
 
+const { config } = require('../../config');
+
```
Comment (hyfi06) [inline components/users/routes.js:9]: you don't use this

File: components/auth/routes.js [hunk 2]
```diff
@@ -49,6 +49,8 @@ function authApi(app) {
             typeOfUser,
           } = user;
 
+          const defaultPath = config.defaultPath[user.typeOfUser] || '/';
+
```
Comment (hyfi06) [inline components/auth/routes.js:53]: In line 22 `next(boom.unauthorized('apiKeyToken is required'));` replace with `throw boom.unauthorized('apiKeyToken is required');`

File: components/users/routes.js [hunk 3]
```diff
@@ -30,8 +32,17 @@ function usersApi(app) {
...
         const user = await usersService.getUserId({ userId });
 
+        const fields = ['password'];
+
+        const data = Object.keys(user)
+          .filter((key) => !fields.includes(key))
+          .reduce((newUser, key) => {
+            newUser[key] = user[key];
+            return newUser;
+          }, {});
```
Comment (hyfi06) [inline components/users/routes.js:42]: `delete user.password`

File: components/auth/routes.js (hunk 2 above)
Comment (ealexisaraujo) [reply components/auth/routes.js:53]: Done!

File: components/users/routes.js (hunk 1 above)
Comment (ealexisaraujo) [reply components/users/routes.js]: done

File: components/users/routes.js (hunk 3 above)
Comment (ealexisaraujo) [reply components/users/routes.js]: 😱

**Pull request merged**